specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json

```diff
@@ -0,0 +1,883 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Monitor Private Link Scopes",
+    "description": "Azure Monitor API reference for Private Links Scopes management.",
+    "version": "2019-10-17-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/microsoft.insights/privateLinkScopes": {
```

Comment:
what effect does this new type have when created?  I.e. when would I use it and what do I expect to happen?  What further setup is required once its created if any?

Reply:
its is a resource responsible of grouping multiple Azure monitor resources. Since a lot of Azure Monitor resources are served via global fqdns, this was required to support private links. so the way it works is as follows:
a. customer creates scope.
b. customer attaches workspace or AI component to the scope (this API is not in this swagger, since might still be subject to minor changes).
c. customer then creates a private link on the scope essentially connecting its VNet to all resources in the scope.